iwf/errors.py

```diff
@@ -112,9 +112,8 @@ def process_workflow_abnormal_exit_error(
 
 
 def parse_unexpected_error(err) -> ErrorResponse:
-    encoding = guess_json_utf(err.content)
-    if encoding is not None:
-        err_dict = jsonlib.loads(err.content.decode(encoding))
-    else:
-        err_dict = jsonlib.loads(err.content)
-    return ErrorResponse.from_dict(err_dict)
```

Comment:
The `guess_json_utf` method is no longer available in `httpx`. It was removed in this commit: https://github.com/encode/httpx/pull/2911/files#diff-33c13e0b177bacd2f02e29bcb8aea5b49e7ce34901fd8f41fefb65defba1bd33

I used the Unit Test in the PR above to create the new solution too.